include/pcl_conversions/pcl_conversions.h

```diff
@@ -39,33 +39,32 @@
 
 #include <vector>
 
-#include <ros/ros.h>
-
 #include <pcl/conversions.h>
 
 #include <pcl/PCLHeader.h>
-#include <std_msgs/Header.h>
+#include <std_msgs/msg/header.hpp>
 
 #include <pcl/PCLImage.h>
-#include <sensor_msgs/Image.h>
+#include <sensor_msgs/msg/image.hpp>
 
 #include <pcl/PCLPointField.h>
-#include <sensor_msgs/PointField.h>
+#include <sensor_msgs/msg/point_field.hpp>
 
 #include <pcl/PCLPointCloud2.h>
-#include <sensor_msgs/PointCloud2.h>
+#include <sensor_msgs/msg/point_cloud2.hpp>
 
-#include <pcl/PointIndices.h>
-#include <pcl_msgs/PointIndices.h>
+// TODO(clalancette): Re-enable once we have pcl_msgs
+//#include <pcl/PointIndices.h>
+//#include <pcl_msgs/PointIndices.h>
 
-#include <pcl/ModelCoefficients.h>
-#include <pcl_msgs/ModelCoefficients.h>
+//#include <pcl/ModelCoefficients.h>
```

Comment:
That's fine, porting `pcl_msgs` should be very straight forward so we can reenable them all at once :+1: